app/views/patient_mailer/assessment_email.html.erb

```diff
@@ -3,7 +3,10 @@
 <% end %>
 
 <% content_for :body do %>
+  <% @count = @patients.count %>
+  <% @i = 0 %>
   <% @patients.each do |patient| %>
```

Comment:
I'm certainly not a ruby expert but could you track the index with `|patient, index|`? and then just check `index` against `@patients.count`?

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/948/commits/68a8452815a1208187bc9e5fda449356e851fa20